work/v1alpha1/types_manifestworkreplicaset.go

```diff
@@ -86,6 +90,23 @@ type LocalPlacementReference struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:MinLength=1
 	Name string `json:"name"`
+
+	// +optional
```

Comment:
Should here add `// +kubebuilder:default={type: All}` ?

Reply:
its already there in the Rollout API definition [here](https://github.com/open-cluster-management-io/api/blob/main/cluster/v1alpha1/types_rolloutstrategy.go#L27)

Reply:
Adding the default value on the top of RolloutStrategy is to ensure even no RolloutStrategy field defined in yaml, 
there's will be RolloutStrategy and "type: All" generated.

Reply:
okay, done.